source/common/buffer/watermark_buffer.h

```diff
@@ -43,6 +43,12 @@ class WatermarkBuffer : public OwnedImpl {
   void appendSliceForTest(const void* data, uint64_t size) override;
   void appendSliceForTest(absl::string_view data) override;
 
+  size_t addFragments(absl::Span<const absl::string_view> fragments) override {
```

Comment:
preference to move to cc file for consistency

Reply:
get it.